### ISSUE TITLE
Stop Multiple uploads iamges

### ISSUE THREAD

**Issue by laurentiustroia** [orthes/medium-editor-insert-plugin#221]
How can stop upload multiple images ? I do a hack to count my image in Core when he press Images button but if he multiple I don't count good, and now for moment best solution is to stop multiple uploads. I know i can use this.core in add function for images but I'm a little confusion about where to do this exactly because i don't use url for upload.

**Comment by orthes** [comment]
No, right now it's not possible to disable multiple image upload.

**Comment by ahmadalukah** [comment]
You can use Jquery-file-upload options in your `fileUploadOptions` : 
```
fileUploadOptions: {
                        singleFileUploads : false
                    }
```
